pyproject.toml

```diff
@@ -22,23 +22,60 @@ classifiers = [
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
 ]
-description = "Extend functionality of `django-import-export`"
-readme = "README.rst"
-license = {file = "LICENSE"}
-keywords = ["django_import_export_extensions"]
-name = "django_import_export_extensions"
-dynamic = ["dependencies", "version"]
 
-[project.urls]
-"Homepage" = "https://github.com/saritasa-nest/django-import-export-extensions"
-"Bug Tracker" = "https://github.com/saritasa-nest/django-import-export-extensions/issues"
+[tool.poetry.dependencies]
+python = ">=3.10, <3.12"
```

Comment:
Please leave a comment why

Reply:
Hm, actually we support 3.12, but we can't run linter checks for it due issue I've described above. I'll extend version to `^3.10`